src/opencensus/proto/trace/v1/trace_config.proto

```diff
@@ -55,12 +55,22 @@ message ProbabilitySampler {
   double samplingProbability = 1;
 }
 
-// Sampler that makes a constant decision (either always "yes" or always "no")
-// on span sampling.
+// Sampler that always makes a constant decision on span sampling.
 message ConstantSampler {
 
-  // Whether spans should be always sampled, or never sampled.
+  // DEPRECATED: whether spans should be always sampled, or never sampled.
   bool decision = 1;
+
+  // How spans should be sampled:
+  // - Always on
+  // - Always off
+  // - Always follow the parent Span's decision (off if no parent).
+  enum ConstantDecision {
+    ALWAYS_OFF = 0;
```

Comment:
What do you think about adding UNSET = 0;

This is for backwards compatibility: if decision_v2 is "unset" (i.e. zero default due to proto3), then decision_v1 takes precedence.